src/richText.js

```diff
@@ -269,6 +269,11 @@ define([
             }
         });
 
+        var STYLE = /<style type="text\/css">.*?<\/style>/g;
+        editor.on('paste', function(event) {
+            event.data.dataValue = event.data.dataValue.replace(STYLE, "");
+        }, null, null, 2 );
```

Comment:
See last comment on http://www.keyvan.net/2012/11/clean-up-html-on-paste-in-ckeditor/ (and my voodoo comment)